Ascend.java

```diff
@@ -2,6 +2,19 @@
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
+	int x = Integer.parseInt(args[0]);

```

Comment:
can use a more describing word for the variable. 
The first argument represents the limit, so maybe call it lim or limit